lambdatimes/src/components/Content/Card.js

```diff
@@ -1,19 +1,33 @@
 import React from 'react';
+import PropTypes  from "prop-types";
+import Cards from "./Cards";
 
 const Card = props => {
   return (
     <div className="card">
-      <div className="headline">{/* headline goes here */}</div>
+      <div className="headline">{props.card.headline}</div>
       <div className="author">
         <div className="img-container">
-          <img src={'' /* image source goes here */} />
+          <img src={props.card.img} />
         </div>
-        <span>By {/* author goes here */}</span>
+        <span>By {props.card.author}</span>
       </div>
     </div>
   );
 };
 
 // Make sure to include PropTypes.
 
+Card.propTypes={
+
+    card:PropTypes.shape({
+        headline:PropTypes.string.isRequired,
+        tab:PropTypes.number,
+        author:PropTypes.arrayOf(PropTypes.string),
```

Comment:
Look at the data types on your props in this component, I think you'll find tab and author are both strings